### DISCUSSION TITLE
博客与写作的一些思考

### DISCUSSION THREAD

**Discussion by shenweiyan** [shenweiyan/Digital-Garden#34] [2.1-乱弹]
关于写作原本的想法，只是想写点自己在学习、工作、生活上的一些心得和体会，把一些知识点记录下来。

留言的初衷是，希望自己写下来的东西能够有人一起探讨。哪怕你把我的观点批评得一无是处也可以，哪怕你说我的句子狗屁不通也没关系，只要有评判，我相信总有一天我会变得更好。

写作也好留言也罢，博客也只是一种实现的载体，语雀、博客园也是同样的道理，平台的选择虽然重要，但根本还是在于内容。

每次看到人家一些内容翔实，界面优雅的博客，总要跟自己说，要忍住再去折腾博客的冲动，什么 UI，什么 CSS/HTML，都是浮云，内容才是核心，老老实实回归最简单的 Issues 和 Discussions 已经足够了。另外，需要知道的是，博客其实就是写给自己看的，什么 SEO 流量、关注度、知名度还是需要一颗随缘的心态。

使用 Issues 当作博客也是挺不错的~

- <https://github.com/Meekdai/Gmeek>
- <https://github.com/yihong0618/gitblog>
- <https://github.com/yutingzhao1991/github-blogs-collector>

关于部分平台商业化的写作，有两点自认为讲的挺好：    

- 商业化已经改变了写作的初衷。    
- 当写文章变成盈利的手段，那么，写出来的东西是没有灵魂的。

**Comment by obaby** [comment]
自娱自乐的小树洞，我也是

**Reply by shenweiyan** [reply]
共勉，你的博客也很好~
